### DISCUSSION TITLE
routing problem

### DISCUSSION THREAD

**Discussion by impossible98** [facebook/docusaurus#9818] [Q&A]
file path: /docs/code/runtime/cgo/cgo.md
but routing must write into `/code/runtime/cgo/`

i wanna `/code/runtime/cgo/cgo`

so how to do it

docusaurus version: v3.1.1

**Comment by slorber** [comment]
https://docusaurus.io/docs/sidebar/autogenerated#category-index-convention

https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#slug
